spec/table_spec.rb

```diff
@@ -2792,6 +2790,24 @@
         end
       end
     end
+
+    context "when `except` is passed" do
+      it 'resizes only the other columns' do
+        # Check that an Array argument works
+        table = Tabulo::Table.new(%w(hello hi there), :to_s, :length, :upcase)
```

Comment:
Please stick to square brackets for `%w[these kinds of arrays]`, for consistency with the project's preexisting style.

Reply:
Done. I also changed an existing example a few lines above where I had picked up these ugly parentheses 😛